README.md

```diff
@@ -21,18 +21,14 @@ The typo3cms binary will be installed in the specified bin-dir (by default `vend
 
 ## Usage
 
-1. You need to activate the extension:
```

Comment:
Why was this step removed?

Reply:
This is a mistake! I will provide an update asap.

Reply:
Update: it is indeed intended, since TYPO3 11 the command is not available anymore.